BottomBar.Droid/Renderers/BottomBarPageRenderer.cs

```diff
@@ -227,7 +231,8 @@ void UpdateBarTextColor ()
 				return;
 			}
 
-			// haven't found yet how to set text color for tab items on_bottomBar, doesn't seem to have a direct way
+            _bottomBar.SetActiveTabColor(Element.BarTextColor.ToAndroid());

```

Comment:
Couldn't you just merge this PR and exclude the project files. I don't know what I could do with the formating as it's formated here